distribution/lib/Standard/Table/0.0.0-dev/src/Data_Formatter.enso

```diff
@@ -207,7 +207,9 @@ type Data_Formatter
     make_decimal_parser self auto_mode=False =
         decimal_point = if self.decimal_point == Auto then Nothing else self.decimal_point
         thousand_separator = if self.thousand_separator.is_empty then Nothing else self.thousand_separator
-        NumberParser.createDecimalParser auto_mode.not (auto_mode.not || self.allow_leading_zeros) self.trim_values self.allow_exponential_notation decimal_point thousand_separator
+        ## NumberParser.createDecimalParser auto_mode.not (auto_mode.not || self.allow_leading_zeros) self.trim_values self.allow_exponential_notation decimal_point thousand_separator
+        ## allowExponentialNotation=self.allow_exponential_notation (ToDo)
```

Comment:
In general we try to keep the format of [ToDo comments](https://github.com/enso-org/enso/blob/develop/docs/style-guide/java.md#todo-comments) as:
```
## TODO message
```
Also the comment above the TODO seems to be quite redundant, shall we remove it?